app/assets/javascripts/discourse/app/templates/review-index.hbs

```diff
@@ -6,8 +6,19 @@
       }}</span>
 
     <ul>
-      {{#each this.unknownReviewableTypes as |type|}}
-        <li>{{type}}</li>
+      {{#each this.unknownReviewableTypes as |reviewable|}}
+        {{#if (eq reviewable.source "unknown")}}
```

Comment:
I have some worry that we are duplicating the raw `unknown` string on both client and server side. What do you think about moving the string into [a JS constant](https://github.com/discourse/discourse/blob/566c772980077f7e2b23e8ef750be6cec5dd2854/app/assets/javascripts/discourse/app/lib/constants.js) whose value is determined by a server side constant?